dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/org/apache/cxf/jaxws/ServiceImpl.java

```diff
@@ -486,6 +564,10 @@ protected <T> T createPort(QName portName, EndpointReferenceType epr, Class<T> s
         if (portInfo != null) {
             clientFac.setBindingId(portInfo.getBindingID());
             clientFac.setAddress(portInfo.getAddress());
+            if (isFinestEnabled) {   // Liberty Change issue #26529
+                LOG.finest("Binding id is set into JaxWsClientFactoryBean(clientFac): " + clientFac.getBindingId());
+                LOG.finest("Address is  set into JaxWsClientFactoryBean(clientFac): " + clientFac.getBindingId());
```

Comment:
I think this should be LOG.finest("Address is  set into JaxWsClientFactoryBean(clientFac): " + clientFac.getAddress());